libmozdata/bugzilla.py

```diff
@@ -78,6 +78,7 @@ def put(self, data, attachment=False, retry_on_failure=False):
         Args:
             data (dict): a dictionnary
         """
+        failures = []
```

Comment:
Nit: Can you use two names which are a bit more different? E.g. to_retry instead of failed.